rubin_scheduler/scheduler/surveys/too_scripted_surveys.py

```diff
@@ -202,19 +202,17 @@ def __init__(
 
         # list to keep track of alerts we have already seen
         self.seen_alerts = []
-
         # Add information for visit metadata if necessary
+        has_tracking_detailer = False
+        for detailer in self.detailers:
+            if isinstance(detailer, TrackingInfoDetailer):
+                has_tracking_detailer = True
```

Comment:
Why move this check earlier and not break out of the loop if one is found? 
I am missing something about why the previous implementation didn't work.

Reply:
I think this was just a readability change I made while debugging. Having a `break` so nested can be tough to parse.